### PR TITLE
CATO-4896V1

### PR DESCRIPTION
**Pull request by rutpatelhmrc**
Changes based on the CATO-4894 PR

### CODE REVIEW

File: src/main/scala/uk/gov/hmrc/ct/computations/CP33.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ import uk.gov.hmrc.ct.computations.retriever.ComputationsBoxRetriever
 case class CP33(value: Option[Int]) extends CtBoxIdentifier(name = "Profit/Losses on disposal of assets") with CtOptionalInteger with Input
   with ValidatableBox[ComputationsBoxRetriever] {
   override def validate(boxRetriever: ComputationsBoxRetriever): Set[CtValidation] = {
-    validateZeroOrPositiveInteger(this)
+    Set()
```
Comment (liquidarmour) [inline src/main/scala/uk/gov/hmrc/ct/computations/CP33.scala]: Why not just remove the function and not implement ValidatableBox?

Comment (rutpatelhmrc) [reply src/main/scala/uk/gov/hmrc/ct/computations/CP33.scala]: I think we just left there so we can test if it accepts the random numbers which we generate as part of our test. So its just there so we can test CP33.

Comment (liquidarmour) [reply src/main/scala/uk/gov/hmrc/ct/computations/CP33.scala]: Why test validation on something that doesn't have any validation?

It also means we are executing a function when we don't need to.

Comment (kastoestoramadus) [reply src/main/scala/uk/gov/hmrc/ct/computations/CP33.scala]: I told the same but they said the tests would serve as documentation according to the convention.

**Pull request merged**